src/main/java/org/spongepowered/api/item/ItemType.java

```diff
@@ -80,5 +82,15 @@
      * @return The item property, if available
      */
     <T extends Property<?, ?>> Optional<T> getDefaultProperty(Class<T> propertyClass);
+    
+    @Override
+    default ItemType getType() {
```

Comment:
Or `getBaseType()`.
Also I'm not sure whether the return type is right.
Maybe this should Optional.empty for BaseTypes.